tests/conftest.py

```diff
@@ -138,9 +138,10 @@ def make_mongodb():
     try:
         subprocess.check_call(cmd, cwd=repo)
     except subprocess.CalledProcessError:
-        print("If on linux or mac, Mongod command failed to execute. If on windows, mongod has not been installed as \n"
-              "a service. In order to run mongodb tests, make sure to install the mongodb community edition\n"
-              "for your OS with the following link: https://docs.mongodb.com/manual/installation/")
+        print("If on linux/mac, Mongod command failed to execute. If on windows, the status of mongo could not be\n"
```

Comment:
"If on..." is a bit confusing.  Maybe make it easier for a non-programmer, something like "If you are using linux or...."